### PR TITLE
Use function hash as workletID instead of a random number.

### PR DESCRIPTION
**Pull request by Szymon20000**
WHY:
Because we want reproducible builds.

HOW:
Use a hash function from https://www.npmjs.com/package/string-hash-64.

REMOVE DEBUG CONSOLE.LOG !!!!!!!!!

### CODE REVIEW

File: plugin.js [hunk 1]
```diff
@@ -145,11 +146,11 @@ function processWorkletFunction(t, fun) {
   const variables = Array.from(closure.values());
 
   const privateFunctionId = t.identifier('_f');
+  const workletID = hash(fun.toString());
```
Comment (kmagiera) [inline plugin.js:149]: can we use the result of `buildWorkletString` call few lines below. I think it makes more sense that we make hash based on the actual content of `asString`

Comment (Szymon20000) [reply plugin.js:149]: We can but it shouldn't matter cause buildWorkletString is the same for identical functions.

Comment (Szymon20000) [reply plugin.js]: Done.

File: plugin.js [hunk 2]
```diff
@@ -149,11 +150,11 @@ function processWorkletFunction(t, fun) {
   // if we don't clone other modules won't process parts of newFun defined below
   // this is weird but couldn't find a better way to force transform helper to
   // process the function
-  const workletID = Math.random() * 1e18;
   const clone = t.cloneNode(fun.node);
   const funExpression = t.functionExpression(null, clone.params, clone.body);
 
   const funString = buildWorkletString(t, fun, variables);
+  const workletID = hash(funString);
```
Comment (kmagiera) [inline plugin.js:157]: can we also now rename the attribute "workletHash" to indicate that this is not an identifier?

Comment (Szymon20000) [reply plugin.js]: Done

**Comment by kmagiera** [comment]
Approved, note that there are some conflicts to be resolved prior to merging

**Pull request merged**